worker/raft/raftforwarder/worker.go

```diff
@@ -127,11 +132,12 @@ func (w *forwarder) processRequest(command []byte) (raftlease.ForwardResponse, e
 		return empty, errors.Trace(err)
 	}
 	respValue := future.Response()
-	responseErr, ok := respValue.(error)
-	if respValue != nil && !ok {
-		return empty, errors.Errorf("FSM response must be an error or nil, got %#v", respValue)
+	response, ok := respValue.(raftlease.FSMResponse)
+	if !ok {
+		return empty, errors.Errorf("expected an FSMResponse, got %#v", respValue)
```

Comment:
Often helpful to include the type %T in the error message

Reply:
Yup